### DISCUSSION TITLE
Change Reynoldsnumber, viscosity for water usage

### DISCUSSION THREAD

**Discussion by x3oo** [ImperialCollegeLondon/sharpy#168] [General]
Is it possible to manipulate the reynoldsnumber, viscosity etc?
I am trying to simulate a hydrofoil.

**Comment by ArturoMS13** [comment]
Hi,
we use potential theory for the fluid dynamics which does not account for viscosity. 
You can find further information [here](https://en.wikipedia.org/wiki/Vortex_lattice_method) or in "J. Katz and A. Plotkin. Low-Speed Aerodynamics. Cambridge University Press, 2001. doi: 10.1017/cbo9780511810329". There you can get a better understanding of the theory and decide if it suits you.

Best regards.

**Reply by x3oo** [reply]
because all my other vlm(xflr5, vspaero and openaerostruct) programs have the option to adjust reynoldsnumber, densitiy and viscosity i was under the impression that it is need for vlm. thank you

**Comment by x3oo** [comment]
i guess xflr5 is using it to estimate parasitic drag etc.
